p0_user_stories.md

```diff
@@ -31,22 +31,22 @@ other pods should be blocked from accessing it via any IP.
 
 - I want all namespaces matching X to be completely 100% locked-down by default; no pods can talk to any other pods until the developers specify policies allowing it - I may also want the converse situation for namespace Y.
 
-- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all my clusters have different Service CIDRs.
+- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all of my clusters have different Service CIDRs.
 
-- I want to block all pods from being able to reach ports on nodes. And the rule should automatically cover all nodes, and should cover all IPs on those nodes, except for port 53.
+- I want to block all pods from being able to reach ports on nodes. The rule should automatically cover all nodes, and all IPs on those nodes, except for port 53.
 
 ## tier-2
 
 - I want all namespaces to be “isolated” (don’t accept traffic from other Namespaces) by default; developers must specify if they want other namespaces to be able to access their services.
 
-- I want to administratively put a choke point between  pods that arent in the same app so i can audit cross-app dependencies and implement ingress controls by default in my cluster.
+- I want to administratively put a choke point between pods that arent in the same app so i can audit cross-app dependencies and implement ingress controls by default in my cluster.
 
 - I want to block all pods from being able to reach the AWS metadata server (169.254.169.254)
 except for pods in Namespace X, except I want to run kube2iam as a metadata proxy, so pods trying to access the metadata server should be redirected to that instead.
 
 - I want all pods to be blocked from accessing the internet.
 
-- I want all pods to be blocked from accessing the internet by default, but developers can add policies declaring what sites they need access to. Then those sites will be reachable, but if hackers break into the pod they can’t reach anywhere else besides those sites, except they can’t access 192.168.0.0/16 even if they declare it.
+- I want all pods to be blocked from accessing the internet by default, but developers can add policies declaring what sites they need access to. The sites will be reachable, but if hackers break into the pod they can’t reach anywhere else besides those sites, except they can’t access 192.168.0.0/16 even if they declare it.
```

Comment:
This story probably needs condensed a bit for increased readability. I would add the change myself, but I am finding it difficult to follow the actual request.

For the first part - Would blocking access to anything not declared in a CIDR egress policy not suffice?